doc/nrf/app_dev/device_guides/nrf54l/fota_update.rst

```diff
@@ -0,0 +1,236 @@
+
+The application supports SMP handlers related to:
+
+* Image management.
+* Operating System (OS) management used to reboot the device after the firmware upload is complete.
+* Erasing settings partition used to ensure that a new application is not booted with incompatible content in the settings partition written by the previous application.
+
+To enable support for FOTA updates, do the following:
+
+* Enable the :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option, which implies configuration of the following:
+
+   * All of the SMP command handlers mentioned in the previous paragraph.
+   * SMP BT reassembly feature.
+   * The :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` Kconfig option automatically extends the Bluetooth buffers, which allows to speed up the FOTA transfer over Bluetooth, but also increases RAM usage.
+
+    .. note::
+       The :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option enables the device to use MCUmgr for performing firmware over-the-air (FOTA) updates using Bluetooth LE.
+       It can be used along with other samples, and is meant as a demonstration of the default DFU configuration over Bluetooth.
+
+.. fota_upgrades_over_ble_intro_end
+
+.. fota_upgrades_over_ble_mandatory_mcuboot_start
+
+* Use MCUboot as the upgradable bootloader (``SB_CONFIG_BOOTLOADER_MCUBOOT`` must be enabled).
+  For more information, go to the :ref:`ug_bootloader_adding_sysbuild_upgradable` page.
+
+.. fota_upgrades_over_ble_mandatory_mcuboot_end
+
+.. fota_upgrades_over_ble_additional_information_start
+
+If necessary, you can modify any of the implied options or defaulted values introduced by the :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option.
+
+You can either add these Kconfig options to the configuration files of your application or have them inline in a project build command.
+Here is an example of how you can build for the :ref:`peripheral_lbs` sample:
+
+.. parsed-literal::
+   :class: highlight
+
+    west build -b *board_target* -- -DCONFIG_BOOTLOADER_MCUBOOT=y -DCONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU=y
+
+When you connect to the device after the build has completed and the firmware has been programmed to it, the SMP Service is enabled with the ``UUID 8D53DC1D-1DB7-4CD3-868B-8A527460AA84``.
+If you want to add SMP Service to advertising data, refer to the :zephyr:code-sample:`smp-svr`.
+
+.. fota_upgrades_over_ble_additional_information_end
+
+.. _ug_nrf54l_developing_ble_fota_mcuboot_kmu:
+
+Provisioning of keys for Hardware KMU
+=====================================
+
+In case of FOTA implementations using the MCUboot bootloader, which includes hardware cryptography and KMU, you must complete key provisioning before booting any application.
+Refer to :ref:`ug_nrf54l_developing_provision_kmu` for detailed description.
+
+
+.. _ug_nrf54l_developing_ble_fota_mcuboot_direct_xip_mode:
+
+Build configuration additions for MCUboot in the direct-xip mode
```

Comment:
Move this section at the end. See [this](https://nordicsemi.atlassian.net/wiki/spaces/UXF/pages/1065583315/2024.10.28_nRF54L15_Production_Launch_Take2#:~:text=The%20paragraph%20%E2%80%9CProvisioning,that%20user%20guide).